lua/graph.lua

```diff
@@ -1404,17 +1406,20 @@ function filter_tags_generic(kv)
    end
 
   --set a few flags
-  local road_class = road_class[kv["highway"]]
+  local rc = road_class[kv["highway"]]
+  if kv["highway"] == "construction" then
+    rc = road_class[kv["construction"]]
```

Comment:
maybe i didnt see this but how do we get away with not checking that `rc` isn't `nil`. it seems to me someone could but a crazy value in there that isnt in our map. maybe we do this on the c++ side somehow or i propbably just missed it

Reply:
if `rc == nil` here, we will set `rc = 7` in the `if-else` blocks later (line 1418-1419)